src/main/java/frc/robot/subsystems/ElevatorSubsystem.java

```diff
@@ -0,0 +1,126 @@
+package frc.robot.subsystems;
+
+import static frc.robot.Constants.ElevatorConstants.*;
+
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.motorcontrol.MotorUtil;
+import frc.robot.shuffleboard.GRTNetworkTableEntry;
+import frc.robot.shuffleboard.GRTShuffleboardTab;
+
+public class ElevatorSubsystem extends SubsystemBase {
+
+    public double speed = 0;
+    private final WPI_TalonSRX motorElevator = MotorUtil.createTalonSRX(MOTORWINCH);
+
+    private final GRTShuffleboardTab shuffleboardTab = new GRTShuffleboardTab("Elevator");
+    private final GRTNetworkTableEntry elevatorPositionEntry = shuffleboardTab.addEntry("ElevatorPosition", 0).at(0, 0);
+
+    public enum Height {
+        GROUND {
+            public Height previous() {
+                return Height.GROUND;
+            }
+            public double get(){
```

Comment:
I suppose this is in the past this year, but for future reference you can use an enum with a private constructor and a value field instead of having to override `.get()` for everything.
```java
public enum Height {
    GROUND(GROUND_HEIGHT), LOW(LOW_HEIGHT), HIGH(HIGH_HEIGHT);

    public double value;
    private Height(double height) {
        this.value = height;
    }
}
```
This is what we did last year for [intake positions](https://github.com/grt192/GRT2022/blob/develop/src/main/java/frc/robot/subsystems/IntakeSubsystem.java#L31-L43).